### DISCUSSION TITLE
Why all the resources have "This resource does not have plugins configured yet. " error message?

### DISCUSSION THREAD

**Discussion by QXgu** [BlueBrain/nexus#2764] [Q&A] [answered]
Hi,

I have a question that all the resources in the project that I created contain the message "This resource does not have plugins configured yet. You can nonetheless edit the resource metadata below." May I know what does not mean? How could I solve this problem?

Thanks,
Amy

**Comment by dhaneshnm** [comment, marked as answer]
Hi Amy,
Fusion (the nexus front end) has a feature called plugins that helps to add custom presentation logic to the resource view. You can find more information about it here, https://bluebrainnexus.io/docs/fusion/plugins.html.
The message you see happens when either deployment has no plugins at all  or no plugin matches the resource type. It is not an error message, and the resource view will work just fine without plugins.

**Reply by QXgu** [reply]
@dhaneshnm Thank you for your help!
